### DISCUSSION TITLE
What exactly does mean "deploying optimizations in Android Studio"?How to turn it off

### DISCUSSION THREAD

**Discussion by H2268** [LSPosed/LSPosed#2839] [Q&A] [answered]
<img width="481" alt="image" src="https://github.com/LSPosed/LSPosed/assets/48410767/5b6e2eed-7a5b-4468-8c61-8744aef19b4c">
I am using "Android Studio Giraffe | 2022.3.1", can you tell me how to turn off deploying optimizations？

**Comment by JasonKhew96** [comment, marked as answer]
in Android Studio, edit your run configuration, tick always install with package manager
OR
use `installDebug`
